Keyboards/KeyboardsBase/KeyboardViewController.swift

```diff
@@ -51,6 +51,11 @@ class KeyboardViewController: UIInputViewController {
     annotationHeight = nounAnnotation1.frame.size.height
 
     loadKeys()
+
+    // Set tap handler for info button on CommandBar
+    commandBar.infoButtonTapHandler = {
+      print("Tapped Info Button!")
```

Comment:
This is obviously only an example as we did not discuss what exactly you want to trigger with the button. :)